components/availability/src/Availability.svelte

```diff
@@ -323,11 +399,82 @@
     .day {
       display: grid;
       grid-template-rows: $headerHeight 1fr;
+      position: relative;
 
       h2 {
         margin: 0;
         padding: 0;
-        text-align: center;
+        display: grid;
+        grid-template-columns: auto 1fr;
+        gap: 0.5rem;
+        height: 30px;
+        line-height: 30px;
+        font-size: 1rem;
+        font-weight: 300;
+
+        .date {
+          border-radius: 15px;
+          background: var(--blue);
+          color: white;
+          font-weight: bold;
+          display: block;
+          width: 30px;
+          height: 30px;
+          line-height: 30px;
```

Comment:
Can this be changed to a number? That way if the page is zoomed in the line-height will scale too. I think this would be `1.875` to get 30px